test_is_prime.py

```diff
@@ -1,8 +1,8 @@
 """
-Testing function that returns whether a number is a prime or not
+Testing function that checks whether a number is prime or not
 """
 
-def is_prime(number):
+def check_prime(number):
```

Comment:
In the test file, you don't want to re-implement the function. Rather, I would just import the function from the is_prime.py file.

You can do this with `from .is_prime import check_prime`

Then write the tests using pytest (instead of print statements) so you get a full summary of failures.

https://docs.pytest.org/en/stable/